docs/docs/solidity_equivalents/addresses.mdx

```diff
@@ -1,3 +1,9 @@
 ---
-title: Addresses (WIP)
+title: Addresses
 ---
+
+Addresses on Starknet are longer than on Ethereum. On Starknet they take up the whole felt space i.e 251 bits.
```

Comment:
bigger instead of longer

Reply:
Can we add information about how the addresses are represented with 256 bits in our flavour of solidity

Reply:
Great point!